sdk/servicebus/service-bus/src/session/messageSession.ts

```diff
@@ -972,6 +972,7 @@ export class MessageSession extends LinkEntity {
   private _deleteFromCache(): void {
     this._receiver = undefined;
     delete this._context.messageSessions[this.sessionId!];
+    delete this._context.expiredMessageSessions[this.sessionId!];
```

Comment:
When SessionLockLostError occurs:
- `onAmqpError` is called which adds an entry for current session in the map
- `onAmqpClose` is called because the receiver is getting closed. 

Can you check if this line https://github.com/Azure/azure-sdk-for-js/blob/%40azure/service-bus_1.0.0-preview.2/sdk/servicebus/service-bus/src/session/messageSession.ts#L398 is hit in this scenario?

If yes, then with the change in this PR, we will be clearing the map

Reply:
Please refer to latest commit, the current changes allow deletion of entry to happen only in non-SessionLockLost error cases and explicit close.

Reply:
About the question, yes - had added in logs and the code path does get hit for when SessionLockLostError occurs. @ramya-rao-a

Reply:
thanks!